### PR TITLE
Add unit test for FeaturesConfig class

### PR DESCRIPTION
**Pull request by creydr**
As per title and as follow up for #3761 

/cc @Cali0707

### CODE REVIEW

File: data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/features/FeaturesConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        Assertions.assertFalse(fc.isAuthenticationOIDC(), "should be false, if feature file is empty");
+
+        try (FileWriter writer =
+                new FileWriter(Paths.get(featuresDir.getAbsolutePath(), FeaturesConfig.KEY_AUTHENTICATION_OIDC)
+                        .toString())) {
+            writer.write("enabled");
+        }
+
+        fc = new FeaturesConfig(featuresDir.getAbsolutePath());
+        Assertions.assertTrue(fc.isAuthenticationOIDC(), "should be false, if feature is enabled");
```
Comment (Cali0707) [inline data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/features/FeaturesConfigTest.java:64]:
```suggestion
        Assertions.assertTrue(fc.isAuthenticationOIDC(), "should be true, if feature is enabled");
```

Comment (creydr) [reply data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/features/FeaturesConfigTest.java]: Copy & paste :D

**Review by Cali0707 — APPROVED** [review]
/lgtm

**Pull request merged**